object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,11 @@
+/* exported defaults */
+
+function defaults(target, source) {
+
+  for (var key in source) {
+    if (!Object.values(target).includes(target[key])) {
```

Comment:
`Object.values` is doing an entire loop over the `target` object every time it is called. This will work, but for large objects, this will slow your function to a crawl. Instead, I would check if `target[key]` is `undefined`, and if it is, use the value from `source` for that key